cmake/SeastarDependencies.cmake

```diff
@@ -141,6 +141,10 @@ macro (seastar_find_dependencies)
   seastar_set_dep_args (rt REQUIRED)
   seastar_set_dep_args (numactl
     OPTION ${Seastar_NUMA})
+  seastar_set_dep_args (Sanitizers
+    COMPONENTS
+      address
+      undefined_behavior)
   seastar_set_dep_args (ucontext REQUIRED)
```

Comment:
Is this complexity typical for projects that use sanitizers? I don't mind merging it, but isn't cmake supposed to do this for us?

The code is incomprehensible to me.

Reply:
i wanted to make it explicit. yes, it's taken care of. see https://github.com/scylladb/seastar/pull/2408#discussion_r1734976089 . but with #2406, we will need to specify it anyway.

Reply:
anyway, i dropped this change in the latest revision.